impress.js

```diff
@@ -178,7 +183,7 @@ const stopApplication = (dir) => {
   }
 };
 
-const stop = async () => {
+async function stop(code = 0) {
```

Comment:
```suggestion
const stop = async (code = 0) => {
```

Reply:
But if we leave the arrow function, then the code will not pass the test. I actually have a question: should we terminate the impress at once? We can have other applications and if one of them has a problem, then in this case we terminate all applications.